### PR TITLE
fix(worker): add metric on the number of msg read per bucket

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

**Comment by jd** [comment]
Actually I think `engine.streams.batch-size` is the same, no?

File: mergify_engine/worker.py [hunk 1]
```diff
@@ -557,6 +557,7 @@ async def _consume_buckets(
             )
 
             messages = await self.redis_stream.xrange(bucket_sources_key)
+            statsd.histogram("engine.buckets.read_size", len(messages))  # type: ignore[no-untyped-call]
```
Comment (sileht) [inline mergify_engine/worker.py:560]: it's the same as engine.streams.batch-size

Comment (sileht) [reply mergify_engine/worker.py:560]: Maybe we should drop engine.streams.batch-size and keep your's !

Comment (sileht) [reply mergify_engine/worker.py:560]: > Actually I think `engine.streams.batch-size` is the same, no?

The only diff is `engine.streams.batch-size` is not computed for pull number 0

Comment (jd) [reply mergify_engine/worker.py:512]: Good point, I've changed it that way.

**Review by sileht — APPROVED** [review]

**Pull request merged**